client/state/current-user/selectors.js

```diff
@@ -58,3 +58,7 @@ export function getCurrentUserCurrencyCode( state ) {
 export function canCurrentUser( state, siteId, capability ) {
 	return get( state.currentUser.capabilities, [ siteId, capability ], null );
 }
+
+export function currentUserHasFlag( state, flagName ) {
```

Comment:
For consistency sake, I think it'd be good to add documentation for this function, like it's been done with all the other functions in this file.